substrate/primitives/staking/src/lib.rs

```diff
@@ -461,6 +499,42 @@ pub struct PagedExposureMetadata<Balance: HasCompact + codec::MaxEncodedLen> {
 	pub nominator_count: u32,
 	/// Number of pages of nominators.
 	pub page_count: Page,
+	/// Number of empty slots in the last page.
+	pub last_page_empty_slots: u32,
```

Comment:
@Ank4n please review the chnages related to this.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6034#discussion_r1831356737